code/game/objects/structures/grille.dm

```diff
@@ -142,6 +142,7 @@
 	return 0
 
 /obj/structure/grille/attackby(obj/item/weapon/W as obj, mob/user as mob)
+	visible_message("<span class='danger'>[user] hits [src] with [W].</span>")
```

Comment:
Do we still care about `\the`? I honestly don't know

Reply:
I think we care in case it makes a difference. Here it wouldn't.

Reply:
There are no cases in which it makes a difference
It is always extraneous, without exception

Reply:
Actually if I don't give a shit and you don't give a shit that's already 50% of active collabs

Reply:
Apparently it *does* make a difference in certain rare cases which I could have sworn I had already tested